integration/test_overload_repro.py

```
@@ -0,0 +1,276 @@
+        for t in query_threads:
+            t.join(timeout=5)
+        write_thread.join(timeout=5)
+
+        info_after = node0.info("MODULES")
+        q_after = info_after.get("search_query_queue_size", 0)
+
+        total_successes = sum(r.get("successes", 0) for r in query_results)
+        total_timeouts = sum(r.get("timeouts", 0) for r in query_results)
+        total_failures = sum(r.get("failures", 0) for r in query_results)
+
+        print(f"\n=== WITH QUEUE DEPTH REJECTION (max=50) ===")
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)